public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -797,7 +797,7 @@ export class PrometheusDatasource
         timeValueTuple.push([timeStampValue, valueValue]);
       });
 
-      const activeValues = timeValueTuple.filter((value) => value[1] >= 1);
+      const activeValues = timeValueTuple.filter((value) => value[1] >= 0);
```

Comment:
@ivanahuckova Is there some context as to why we were filtering out values less than 1?

Reply:
Oops, I had a bug in here, we didn't want to include 0 values